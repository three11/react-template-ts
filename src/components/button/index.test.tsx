import * as React from 'react';
import { render } from '@testing-library/react';

import { Button } from '.';

test('should render successfully', () => {
	const { asFragment } = render(<Button />);

	expect(asFragment()).toMatchSnapshot();
	expect(true).toBeTruthy();
});
