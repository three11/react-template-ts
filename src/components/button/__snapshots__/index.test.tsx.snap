// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render successfully 1`] = `
<button
  className="c-btn "
/>
`;
